french/feu-de-camp.md

```diff
@@ -32,4 +32,6 @@ Que faites-vous ?
 
 [Je me lève et me dirige vers la sortie](sortie/pousser-porte/pousser-porte.md)
 
-[Je ne fais rien](rien/rien.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.